presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTypeHandle.java

```diff
@@ -48,7 +49,7 @@ public int getJdbcType()
     }
 
     @JsonProperty
-    public String getJdbcTypeName()
+    public Optional<String> getJdbcTypeName()
```

Comment:
you could change toString to use `jdbcTypeName.orElse(null)` or `jdbcTypeName.orElse("")`